docs/pages/feature-matrix.mdx

```diff
@@ -13,91 +13,184 @@ ensure that all tables are aligned.*/}
+server, database and other Teleport Protected Resources to gain visibility and
+control of all your infrastructure.
+
+|  | **Teleport Enterprise (Cloud)** | **Teleport Enterprise (Self-Hosted)** | **Teleport Community Edition** |
+|---|:---:|:---:|:---:|
+| Protecting:  [Applications](https://goteleport.com/docs/enroll-resources/application-access/getting-started/),  [Databases](https://goteleport.com/docs/enroll-resources/database-access/getting-started/), [Kubernetes Clusters](https://goteleport.com/docs/enroll-resources/kubernetes-access/getting-started/), [Linux Servers](https://goteleport.com/docs/enroll-resources/server-access/getting-started/), [Windows Servers](https://goteleport.com/docs/enroll-resources/desktop-access/introduction/), [Windows Desktops](https://goteleport.com/docs/enroll-resources/desktop-access/introduction/), Cloud Consoles & Resources (AWS, Azure, GCP), [GitHub](https://goteleport.com/docs/admin-guides/management/guides/github-integration/) | ✔ | ✔ | ✔ (does not include Oracle support) |
+
+**Secure remote access:** Zero-trust, auditable  access to all supported
+infrastructure  types from anywhere, without VPNs, bastion hosts or Vaults with
+shared secrets.
+
+|  | **Teleport Enterprise (Cloud)** | **Teleport Enterprise (Self-Hosted)** | **Teleport Community Edition** |
+|---|:---:|:---:|:---:|
+| Dynamic, self-updating inventory | ✔ | ✔ | ✔ |
+| Supports SSH, RDP, Kubernetes, Databases, AWS, Azure, GCP API and CLI, Web applications and services, TCP endpoints for Linux, Windows and MacOS. | ✔ | ✔ | ✔ |
+| [Machines](https://goteleport.com/docs/enroll-resources/machine-id/) & [workloads](https://goteleport.com/docs/enroll-resources/workload-identity/introduction/#teleport-workload-identity-vs-machine-id) | Available in Teleport Machine & Workload Identity |  |  |
+| Agentless Integration with [OpenSSH Servers](https://goteleport.com/docs/enroll-resources/server-access/openssh/openssh-agentless/) | ✔ | ✔ | ✔ |
+| [IP-Based Restrictions](https://goteleport.com/docs/admin-guides/access-controls/guides/ip-pinning/) | ✔ | ✔ | ✖ |
+| [Teleport VNet](https://goteleport.com/docs/connect-your-client/vnet/) | ✔ | ✔ | ✔ |
+
+**Short-lived privileges:** Ephemeral authorization granted through short-lived
+certificates bound to biometric devices and secure-enclaves, via Teleport’s
+built-in certificate authority for X.509 and SSH certificates for all resources,
+including legacy systems.
+
+|  | **Teleport Enterprise (Cloud)** | **Teleport Enterprise (Self-Hosted)** | **Teleport Community Edition** |
+|---|:---:|:---:|:---:|
+| [Role-Based Access Control](https://goteleport.com/docs/admin-guides/access-controls/guides/role-templates/) | ✔ | ✔ | ✔ |
+| [Just-in-Time Access Requests & Reviews](https://goteleport.com/docs/admin-guides/access-controls/access-requests/resource-requests/) | Available in Teleport Identity Governance |  | Only can request roles through CLI  |
```

Comment:
Let's repeat the "Available in..." text for both Cloud and Self-hosted here, too